app/controllers/admin/facets_controller.rb

```diff
@@ -35,8 +35,44 @@ def destroy
     redirect_to admin_facets_path
   end
 
+  def remove_item
+    begin
+      set_visibility
```

Comment:
I should add - if we go with "toggle_item", we'd use it to replace both "add_item" and "remove_item"